src/test/unit/manipulation-test-helper.js

```diff
@@ -145,6 +156,16 @@ function manipulationTestHelper(a) {
 
     function manipulateAndAssert(manipulationCause) {
 
+        if (
+            skipKeyboardShortcutTests &&
+            manipulationCause === EXECUTE.KEY_COMBO
+        ) {
+            if (expect() === null) {
+                expect(0);
```

Comment:
@winhamwr so please merge #659 first, then review this and don't merge—let me fix this and then I'll merge.